particles/ProductMultiplexer/ProductMultiplexer.manifest

```diff
@@ -0,0 +1,20 @@
+# @license
+# Copyright (c) 2017 Google Inc. All rights reserved.
+# This code may only be used under the BSD style license found at
+# http://polymer.github.io/LICENSE.txt
+# Code distributed by Google as part of this project is also
+# subject to an additional IP rights grant found at
+# http://polymer.github.io/PATENTS.txt
+
+import '../../entities/Product.manifest'
+
+shape HostedParticleShape
+  HostedParticleShape(in Product foo)
+  consume annotation
+
+# TODO: This particle should use generic handle type and slot name.
+particle ProductMultiplexer in 'ProductMultiplexer.js'
+  ProductMultiplexer(host HostedParticleShape hostedParticle, in [Product] products)
+  consume set of annotation
+    provide annotation
```

Comment:
I'm not sure that this should provide annotation here - won't that make a singleton annotation slot externally available?

Reply:
right, it is weird for a set slot to provide a singleton slot anyway, 
and it is already experessed in the shape, that the hosting particle will consume an "annotation" slot
thanks for noticing!

PS: one thing that is not expressed, is that the provided slot is designated for the products handle. but i think it's still ok.